plugin/src/test/scala/com/typesafe/genjavadoc/SignatureSpec.scala

```diff
@@ -132,6 +139,7 @@ class SignatureSpec extends WordSpec with Matchers {
         c.getDeclaredMethods.filterNot(x ⇒ filter && (defaultFilteredStrings.exists { s => x.getName.contains(s) }
           || javaKeywords.contains(x.getName)
           || x.getName == "$init$" // These synthetic methods show up in 2.12.0-M4+ even though they are not in the generated Java sources
+          || (filter && Modifier.isStatic(x.getModifiers) && x.getName.endsWith("$")) // These synthetic static methods appear since 2.12.0-M5+ as companions to default methods
```

Comment:
I'm not entirely sure if this is correct though.
I'm filtering test results here, perhaps these methods should never appear in javadoc instead though? cc @2m 

I was not able to figure out where they come from though :(